web/impact/impact/v1/views/user_list_view.py

```diff
@@ -121,7 +160,25 @@ def _serialize_user(user):
         "last_name": user.short_name,
         "is_active": user.is_active,
         "gender": user_gender(user),
+        'updated_at': _get_profile_update_timestamp(user)
+    }
+
+
+def _get_profile_class(user):
+    if BaseProfile.objects.filter(user=user).exists():
+        base_profile = BaseProfile.objects.get(user=user)
+        profile_types = {
+            'EXPERT': ExpertProfile,
+            'ENTREPRENEUR': EntrepreneurProfile,
+            'MEMBER': MemberProfile
         }
+        return profile_types.get(base_profile.user_type)
+
+
+def _get_profile_update_timestamp(user):
+    profile_class = _get_profile_class(user)
```

Comment:
Seems like this should use `impact.utils.get_profile` rather than essentially reinventing it with _get_profile_class.